packages/graph-explorer/src/modules/NodesStyling/NodeStyleDialog.tsx

```diff
@@ -233,9 +205,9 @@ function Content({ vertexType }: { vertexType: string }) {
           <ColorInput
             label="Color"
             labelPlacement="inner"
-            color={localColors.color}
+            color={vertexStyle?.color || "#17457b"}
             onChange={(color: string) =>
-              setLocalColors(prev => ({ ...prev, color }))
+              startTransition(() => setVertexStyle({ color }))
```

Comment:
I remember I added the `startTransition` wrapper, but I don't know if I ever tested to see if it was helping the responsiveness. If it turns out it is not providing any benefit, let's remove it. If it is improving the responsiveness, then let's add it to the other color inputs.